docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+      start demonstration installation guide which is not appropriate for
+      practical production usage. You may want to customize your deployment to
+      one which is more suited towards your specific needs. To customize your
+      deployment you could override the stock values in the values.yaml file,
+      as in the following example:</para>
+
+      <programlisting>helm install myhpcc hpcc/hpcc -f myoverrides.yaml</programlisting>
+
+      <para>The above example uses the <emphasis>myoverrides.yaml</emphasis>
+      file via the -f parameter, which overrides the specified values in the
+      HPCC Systems values.yaml file. It's important to note that this merges
+      the overrides from myoverrides.yaml. Anything that's in the values in
+      the helm chart itself that is not overwritten by the passed in values
+      will remain active.</para>
+
+      <para>A customized deployment can be done using a separate yaml file,
+      such as the <emphasis>myoverrides.yaml</emphasis> used in the above
+      example and merged using helm. Further information about a customized
+      deployments is covered in other sections, as well as the Kubernetes Helm
+      documentation. Consulting the Helm documentation provides complete
+      detail for every aspect of Helm chart usage, and not only for a few
+      select cases described.</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>To (set a value to) disable something in a values file, it
+          won't add that component without first removing the entry from the
+          values file.</para>
+        </listitem>
+
+        <listitem>
+          <para>You can supply multiple values files and they can be
+          combined.</para>
+        </listitem>
+
+        <listitem>
+          <para>When there are 2 yaml files, if there is a matching entry
+          (anything other than a dictionary) the value from 2nd file will
+          overwrite the first.</para>
+        </listitem>
+
+        <listitem>
+          <para>Dictionaries will always be merged.</para>
+        </listitem>
+      </itemizedlist>
+
+      <para>For instance, you want to update logging detail. You could have
+      another file to update that value, or any other list value using an
+      override yaml file. That override yaml file would be considered a second
+      file in addition to the values.yaml file.</para>
+
+      <para>As we will see later, components are defined as lists, so any
+      definition of a component in a user values file will replace all
+      instances of the component in the default chart. Other options (for
+      instance configuring the costs for cpu or file access) are implemented
+      as a dictionary, so options can be selectively set in a users values
+      file, and the other options will be retained.</para>
+
+      <sect3 id="merging_AND_Overrides">
+        <title>Merging and Overriding</title>
```

Comment:
A chunk of the previous section is about merging as well - should be combined/rationalised.